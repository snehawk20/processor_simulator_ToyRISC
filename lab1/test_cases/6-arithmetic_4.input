	.data
a:
	10
d:
	-3
n:
	6
