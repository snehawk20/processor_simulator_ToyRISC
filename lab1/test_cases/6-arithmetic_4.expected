65530		: -5
65531		: -2
65532		: 1
65533		: 4
65534		: 7
65535		: 10
